examples/k8s/README.md

```diff
@@ -0,0 +1,30 @@
+# Kubernetes executor
+
+## Getting started
+
+To get started install and configure a local Kubernetes cluster.
+
+On Mac, you can install Kubernetes with homebrew:
+
+```sh
+brew install kubernetes
+```
+
+The cluster can be started with the command:
+
+```sh
+minikube start
+```
+
+Now we need to give out Kubernetes cluster permission to pull Docker images from ECR:
+
+```sh
+ACCOUNT=$(aws ecr describe-registry --query registryId --output text)
```

Comment:
This was the missing trick to get the cluster to work with AWS services such as ECR for pulling Docker images.